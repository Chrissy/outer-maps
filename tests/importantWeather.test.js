import React from 'react';
import ImportantWeather from '../components/importantWeather.js';
import renderer from 'react-test-renderer';

const props = {
  maxTemperature: 100,
  minTemperature: 0,
  chanceOfPercipitation: 12
}

it('renders', () => {
  const tree = renderer.create(<ImportantWeather {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
