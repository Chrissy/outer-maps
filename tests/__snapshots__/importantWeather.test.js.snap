// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <div
    className="label"
  >
    Average weather this week
  </div>
  <div
    className="importantWeather"
  >
    <div
      className="importantWeatherUnit high hot"
    >
      <div
        className="flex"
      >
        <test-file-stub
          className="icon"
        />
        <div
          className="data"
        >
          100
          °
        </div>
      </div>
      <div
        className="label"
      >
        High Temperature
      </div>
    </div>
    <div
      className="importantWeatherUnit low cold"
    >
      <div
        className="flex"
      >
        <test-file-stub
          className="icon"
        />
        <div
          className="data"
        >
          0
          °
        </div>
      </div>
      <div
        className="label"
      >
        Low Temperature
      </div>
    </div>
    <div
      className="importantWeatherUnit precip"
    >
      <div
        className="flex"
      >
        <test-file-stub
          className="icon"
        />
        <div
          className="data"
        >
          1%
        </div>
      </div>
      <div
        className="label"
      >
        Chance percipitation
      </div>
    </div>
  </div>
</div>
`;
